homeassistant/components/alarm_control_panel/manual.py

```diff
@@ -185,3 +187,13 @@ def _validate_code(self, code, state):
         if not check:
             _LOGGER.warning("Invalid code given for %s", state)
         return check
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes."""
+        state_attr = super().state_attributes
```

Comment:
Create a new dictionary instead. The entity base class will take care of merging state attributes with device state attributes.